FizzBuzz - single parameter.rb

```diff
@@ -0,0 +1,50 @@
+def fizz_buzz
+
+  puts "Pick a number:"
+  is_fb = gets.chomp
```

Comment:
we could use a more intuitive name than `is_fb`for input from the user. It's getting a little bit confusing with `is_fb` and `is_float` next to each other

Reply:
First thing that comes to mind is `input` - or would it be too vague?

Reply:
input is perfectly fine